src/features/components/templates/navigation/affiliates-nav/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import Link from 'features/components/atoms/link'
+import Image from 'features/components/atoms/image'
+import NavTemplate from 'features/components/templates/navigation/template'
+import LanguageSwitcher from 'features/components/molecules/language-switcher'
+import { partners_nav_logo } from 'features/components/templates/navigation/payment-agent-nav/payment-agent-nav.module.scss'
+import PartnerNavLogo from 'images/svg/partner-nav-logo.svg'
+
+const AffiliateNav = () => {
+    return (
+        <NavTemplate
+            renderLogo={() => (
+                <Link url={{ type: 'internal', to: '/partners/' }}>
+                    <Image
+                        src={PartnerNavLogo}
+                        alt={'nav affiliates'}
```

Comment:
Why not `alt="nav affiliates"`?

Reply:
as far as I know it's the same, isn't it? I can change it everywhere if needed

Reply:
its the same, just autocomplete